stdlib/public/core/UnsafePointer.swift.gyb

```diff
@@ -423,34 +417,25 @@ extension ${Self} : CustomDebugStringConvertible {
   }
 }
 
-${MirrorDecl} {
-  ${MirrorBoilerplate}
-
-  var count: Int { return 1 }
-
-  func _getPointerValue() -> UInt64 {
-    return UInt64(Int(Builtin.ptrtoint_Word(_value._rawValue)))
-  }
-
-  subscript(i: Int) -> (String, _MirrorType) {
-    switch i {
-    case 0: return ("pointerValue",_reflect(_getPointerValue()))
-    default: _preconditionFailure("cannot extract this child index")
-    }
+extension ${Self} : CustomReflectable {
+  public func customMirror() -> Mirror {
+    let ptrValue = UInt64(Int(Builtin.ptrtoint_Word(_rawValue)))
+    return Mirror(self, children: ["pointerValue": ptrValue])
   }
+}
 
+extension ${Self} : CustomPlaygroundQuickLookable {
   var summary: String {
     let selfType = "${Self}"
-    let ptrValue = _getPointerValue()
-    if ptrValue == 0 { return "\(selfType)(nil)" }
-    return "\(selfType)(0x\(_uint64ToString(ptrValue, radix:16, uppercase:true)))"
+    let ptrValue = UInt64(Int(Builtin.ptrtoint_Word(_rawValue)))
```

Comment:
The old code was wrong for large pointer values (for `> Int64.max` it would crash), let's fix it:

`UInt64(bitPattern: Int(...))`